src/components/sections/contest/ContestDetails/ContestOverview.tsx

```diff
@@ -59,7 +60,16 @@ const ContestOverview = ({contest, myProject}: ContestOverviewProps) => {
         )}
       </div>
       <div className="mt-12 grid grid-cols-4 grid-rows-2 gap-6">
-        {contest.endDate < new Date() ? (
+        {DateTime.fromJSDate(contest.startDate) < DateTime.now() &&
+        DateTime.fromJSDate(contest.endDate) > DateTime.now() &&
+        contest.status === ContestStatus.APPROVED ? (
```

Comment:
You could consider extracting this into `isContestRunning` variable for readability.